packages/chatbot-server-mongodb-public/src/tracing/scrubbedMessages/makeScrubbedMessagesFromTracingData.ts

```diff
@@ -64,9 +65,9 @@ export async function makeScrubbedMessagesFromTracingData({
     createdAt: assistantMessage.createdAt,
     customData: assistantMessage.customData,
     request: {
-      userTopics: userAnalysis?.topics,
-      origin: userMessage?.customData?.origin,
-      originCode: userMessage?.customData?.originCode,
+      userTopics: userAnalysis?.topics ?? undefined,
```

Comment:
```suggestion
      userTopics: userAnalysis?.topics,
```

The optional chaining operator (`?.`) implicitly evaluates to `undefined` if `userAnalysis` is `undefined | null`, so we don't need this extra use of the nullish coalescing operator (`??`) unless we want to default to something other than `undefined`.